rosie/chamber_of_deputies/tests/test_election_expenses_classifier.py

```diff
@@ -9,18 +9,27 @@
 class TestElectionExpensesClassifier(TestCase):
 
     def setUp(self):
-        self.dataset = pd.read_csv('rosie/chamber_of_deputies/tests/fixtures/election_expenses_classifier.csv',
-                                   dtype={'name': np.str, 'legal_entity': np.str})
-        self.subject = ElectionExpensesClassifier()
+        self.election_expenser_classifier = ElectionExpensesClassifier()
 
-    def test_is_election_company(self):
-        self.assertEqual(self.subject.predict(self.dataset)[0], True)
+    def test_legal_entity_is_a_election_company(self):
+        self.dataframe = pd.read_csv('rosie/chamber_of_deputies/tests/fixtures/election_expenses_classifier.csv',
```

Comment:
Hey @anaschwendler @cuducos 

How could I create a data frame here from a string or a data structure? Just to avoid have tests accessing file system and be able to be more specific with just what is necessary for this test.
Something like: `pd.read('just the information necessary for this test)`

Reply:
A dataframe is a structure of rows and columns. So something like `pd.DataFrame(data=[['my text']])` might work (the first list is the row, the inside list is the single column I guess).

Reply:
Nice with your tip and some search I was able to do what I was planing. :)